command.py

```diff
@@ -66,15 +66,15 @@
 }
 
 multipart_command_types = {
-    'new_waypoint':             {'cmd':128, 'type':'ddffB'},
-    'insert_waypoint':          {'cmd':129, 'type':'ddffBBB'},
-    'set_returnHomeCoordinates':{'cmd':130, 'type':'ddf'},
-    'tare_IMU':                 {'cmd':131,  'type':'fff'},
-    'set_IMU':                  {'cmd':132,  'type':'fff'},
-    'set_KDValues':              {'cmd':133,  'type':'fffffff'},
-    'set_KPValues':              {'cmd':134,  'type':'fffffff'},
-    'set_KIValues':              {'cmd':135,  'type':'fffffff'},
-    'update_waypoint':          {'cmd':136, 'type':'ddffBB'},
+    'new_waypoint':             {'cmd':128, 'type':'ddffB'}, # lon, lat, alt, rad, type
+    'insert_waypoint':          {'cmd':129, 'type':'ddffxBB'}, # lon, lat, alt, rad, prev, next
```

Comment:
The ground station gives it ddffBB, so the padding make sure the data gets sent in the right place. All of the data is parsed on the struct by the same waypointWrapper, so the paddings allows all waypoint commands to be stored in the same struct.